www/lobby/:id.ts

```diff
@@ -15,15 +17,39 @@ lobby_id_router.post('/:id', async (req: Request, res: Response) => {
  * Body Params: id, refreshToken
  */
 lobby_id_router.get('/:id', async (req: Request, res: Response) => {
-  res.status(200).json('placeholder get');
+  const lobbyId = req.params.id;
+  const userId = req.body.id;
+  const lobby = await Lobby.fromId(lobbyId);
+  if (!lobby) return res.status(404).json('Lobby not found in database').end();
+
+  const user = await User.fromId(userId);
+  if (!user) return res.status(404).json('User not found in database').end();
```

Comment:
this is duplicated for each of the routes b/c we need to make sure the lobbyId/userIds are valid and get the objects from the ids but I'm not sure what the best way to get rid of this is

Reply:
using middleware would work but I don't think you can pass objects between handlers so each handler would still  need to get User.fromId and Lobby.fromId

Reply:
How about we just make it a function?